### PR TITLE
pytrainer: 2.0.1 -> 2.0.2

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/pytrainer/default.nix [hunk 1]
```diff
@@ -1,81 +1,76 @@
...
+    matplotlib
+    lxml
     setuptools
+    requests
+    gdal
   ];
 
   nativeBuildInputs = [
     gobject-introspection
     wrapGAppsHook
```
Comment (jtojnar) [inline pkgs/applications/misc/pytrainer/default.nix:50]: Suprised this works without `strictDeps = false`

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 117500` at 0b575619 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/f26eaccac27354e92a9cb93cb1916f19">pytrainer</a></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `gnome3`.
Near pkgs/applications/misc/pytrainer/default.nix:5:3:
```
  |
5 | , gnome3
  |   ^
```
</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement.

File: pkgs/applications/misc/pytrainer/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/applications/misc/pytrainer/default.nix:50]: that is the default if it is not set to true

Comment (jtojnar) [reply pkgs/applications/misc/pytrainer/default.nix:50]: Not in `buildPythonApplication`, unless it changed.

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Pull request merged**